src/test/regress/sql/multi_create_fdw.sql

```diff
@@ -12,5 +12,34 @@ RETURNS fdw_handler
 AS 'citus'
 LANGUAGE C STRICT;
 
+set citus.enable_ddl_propagation to off;
 CREATE FOREIGN DATA WRAPPER fake_fdw HANDLER fake_fdw_handler;
 CREATE SERVER fake_fdw_server FOREIGN DATA WRAPPER fake_fdw;
+set citus.enable_ddl_propagation to on;
+
+-- test propagating foreign server creation
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER foreign_server
+        FOREIGN DATA WRAPPER postgres_fdw
+        OPTIONS (host 'testhost', port '5432', dbname 'testdb');
+
+SELECT COUNT(*)=1 FROM pg_foreign_server WHERE srvname = 'foreign_server';
+\c - - - :worker_1_port
+-- verify that the server is created on the worker
+SELECT COUNT(*)=1 FROM pg_foreign_server WHERE srvname = 'foreign_server';
+\c - - - :master_port
+ALTER SERVER foreign_server OPTIONS (ADD passfile 'to_be_dropped');
+ALTER SERVER foreign_server OPTIONS (SET host 'localhost');
+ALTER SERVER foreign_server OPTIONS (DROP port, DROP dbname);
+ALTER SERVER foreign_server OPTIONS (ADD port :'master_port', dbname 'regression', DROP passfile);
+ALTER SERVER foreign_server RENAME TO foreign_server_1;
+\c - - - :worker_1_port
+-- verify that the server is renamed on the worker
+SELECT srvoptions FROM pg_foreign_server WHERE srvname = 'foreign_server_1';
+\c - - - :master_port
+
+DROP SERVER foreign_server_1;
+\c - - - :worker_1_port
```

Comment:
Can you add some tests to show how DDL commands we've added works on worker nodes. Those tests will make more sense when we start to propagate pg_dist_object tuples together with https://github.com/citusdata/citus/pull/5415